timetracker/models.py

```diff
@@ -1,31 +1,43 @@
 from django.db import models
 from django.utils import timezone
+from datetime import date
 
 
 # Create your models here.
 
 class Employee(models.Model):
-    name = models.CharField(max_length=50)
+    name = models.CharField(max_length=50, blank=False)
 
     def __str__(self):
         return self.name
 
+
 class Checking(models.Model):
     checkChoices = [
         ('in', 'Check-in'),
         ('out', 'Check-out')
     ]
-    userid = models.ForeignKey(Employee, on_delete=models.CASCADE)
-    check = models.CharField(max_length=3, choices=checkChoices)
+    employee = models.ForeignKey(Employee, on_delete=models.CASCADE)
+    check = models.CharField(max_length=3, choices=checkChoices, blank=False)
     time = models.DateTimeField(default=timezone.now)
 
     def __str__(self):
-        return self.userid, self.check, self.time
+        return self.employee.name, self.check
+
+
+class WorkingHours(models.Model):
```

Comment:
Instead of having this extra model, can you please update the api so it doesn't accept a checkin until there is a checkout for the previous checkin.